### ISSUE TITLE
So how do I use this with the openmsp distribution?

### ISSUE THREAD

**Issue by salmansheikh** [speters/openmspde0#1]
Do I just put in a directory and compile these sources files so it works on a DE0 Nano? Not clear to me.

**Comment by speters** [comment]
See https://github.com/speters/openmspde0/blob/master/openMSP430_fpga_top.vhd on how the openMSP verilog modules are instantiated in VHDL.

Keep in mind that this is not a completed project, more a playground.

**Comment by salmansheikh** [comment]
I got it compiled and programmed but the openmsp430 loader is giving me
errors about a cpu_id not valid or a serial port...not sure which. I am
using cygwin tcsh, not sure of that is the problem. I will install
activestate TCL and see if that fixes things.

**Comment by salmansheikh** [comment]
I got it compiled and programming but my memledtest.elf is for another
board...Do you have any c program to test this...

On Tue, Jun 2, 2015 at 5:44 AM, Sönke J. Peters notifications@github.com
wrote:

> See
> https://github.com/speters/openmspde0/blob/master/openMSP430_fpga_top.vhd
> on how the openMSP verilog modules are instantiated in VHDL.
> 
> Keep in mind that this is not a completed project, more a playground.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/speters/openmspde0/issues/1#issuecomment-107897761.

## 

Even a Smile is charity :)
- Prophet Muhammad

**Comment by speters** [comment]
The sources of `memledtest` are in the original OpenMSP430 repository at [/openmsp430/trunk/fpga/altera_de1_board/software/memledtest/](http://opencores.org/websvn,listing?repname=openmsp430&path=%2Fopenmsp430%2Ftrunk%2Ffpga%2Faltera_de1_board%2Fsoftware%2Fmemledtest%2F#path_openmsp430_trunk_fpga_altera_de1_board_software_memledtest_)

**Comment by salmansheikh** [comment]
Yes but that doesn't work for the de0 without modifications as the hardware
is different.
